src/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Superhero",
+  "name": "Superhero Wallet",
   "description": "Superhero Wallet",
```

Comment:
Maybe we can update this description to not be the same as the name? @onvisions

Reply:
I have provided meta description suggestion long ago asked by @CedrikNikita.Actually we even have a backlog task for this one and it has been approved by @smaroudasunicorn. I guess it makes sense to use the same here. 
It was as follows: "Superhero is a multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Powered by æternity."

Reply:
> Superhero is a multi-blockchain wallet...

But the name of the app is "Superhero Wallet". In other hand we would have this weird sentence "Superhero Wallet is a multi-blockchain wallet...". So maybe we could go without "Superhero is"? Ten the description would look as follows:

> Multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Powered by æternity.

@onvisions @smaroudas

Reply:
Sure. I agree.

Reply:
This PR is about changing Superhero to Superhero Wallet in the stores. From now on the brand name of our app is Superhero Wallet and not Superhero. We can create another task for welcome page changes.